packages/smithy-core/src/smithy_core/interfaces/exceptions.py

```diff
@@ -10,4 +12,4 @@ class HasFault(Protocol):
     All modeled errors will have a fault that is either "client" or "server".
     """
 
-    fault: ClassVar[Literal["client", "server"]]
+    fault: Fault | None
```

Comment:
is the `| None` needed here if `Fault` is already defined as `Literal["client", "server"] | None`?